net.sourceforge.liferea.json

```diff
@@ -0,0 +1,63 @@
+{
+    "app-id":"net.sourceforge.liferea",
+    "runtime":"org.gnome.Platform",
+    "runtime-version":"3.36",
+    "sdk":"org.gnome.Sdk",
+    "command":"liferea",
+    "cleanup":[
+        "/include",
+        "/lib/pkgconfig",
+        "/share/pkgconfig",
+        "/share/aclocal",
+        "*.la",
+        "*.a"
+    ],
+    "finish-args":[
+        "--device=dri",
+        "--share=ipc",
+        "--socket=pulseaudio",
+        "--socket=x11",
+        "--socket=wayland",
+        "--share=network",
+        "--talk-name=org.freedesktop.secrets"
+    ],
+    "modules":[
+        {
+            "name":"libpeas",
+            "buildsystem" : "meson",
+            "cleanup":[
+                "/bin/*",
+                "/lib/peas-demo"
+            ],
+            "sources":[
+                {
+                    "type":"archive",
+                    "url":"https://download.gnome.org/sources/libpeas/1.28/libpeas-1.28.0.tar.xz",
+                    "sha256":"42d91993b46ed50f16add6d9577ecc22beb8e2dffa7101e2232c2b63733b8b15"
+                }
+            ]
+        },
+        {
+            "name":"intltool",
+            "sources":[
+                {
+                    "type":"archive",
+                    "url":"https://launchpad.net/intltool/trunk/0.51.0/+download/intltool-0.51.0.tar.gz",
+                    "sha256":"67c74d94196b153b774ab9f89b2fa6c6ba79352407037c8c14d5aeb334e959cd"
+                }
+            ]
+        },
+
+        {
+            "name":"liferea",
+            "sources":[
+                {
+                    "type":"git",
+                    "url":"https://github.com/mikelolasagasti/liferea.git",
```

Comment:
I understand this is your personal fork. Have you contacted upstream developers about submitting their app to Flathub?

Reply:
Correct. It is easier to test patches this way than manually adding patches as source.

And yes, I contacted Lars from Liferea about this request. My plan is to give him and whoever project requires access to resultant flathub repository.

Reply:
All fine from upstream :-)